src/components/storage/ShoppingCartTableSkeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import { Skeleton } from '@/components/ui/Skeleton';
+
+export default function ShoppingCartTableSkeleton() {
+  return (
+    <>
+      <div className='mt-4 grid grid-cols-10 gap-2'>
+        <Skeleton className='col-span-1 h-5' />
+        <Skeleton className='col-span-3 h-5' />
+        <Skeleton className='col-span-5 h-5' />
+        <Skeleton className='col-span-1 h-5' />
+      </div>
+      {Array.from({ length: 4 }).map((_, index) => {
+        return (
+          // biome-ignore lint/suspicious/noArrayIndexKey: Skeleton items don't have a unique id to use.
```

Comment:
instead of commenting out the lint error you can jsut give it a key that is a string + the index. Using a translation preferably. Pretty sure that was don somewhere in news but maybe i am confusing it with another project